packages/vue-puppeteer/package.json

```diff
@@ -10,11 +10,11 @@
     "express": "4.x",
     "express-static": "1.x",
     "mocha": "latest",
-    "puppeteer": "^19.6.2",
+    "puppeteer": "^24.4.0",
```

Comment:
Puppeteer <22 is deprecated, figured we'd move to latest

Reply:
Most recent PR (#114) is actually updating puppeteer for the `puppeteer` folder, this is the `vue-puppeteer` folder (I was confused)